ui/component/channelSections/SectionList/index.js

```diff
@@ -20,7 +21,7 @@ const select = (state, props) => {
     myUnpublishedCollections: selectMyUnpublishedCollections(state),
     myEditedCollections: selectMyEditedCollections(state),
     isPublishing: selectFeaturedChannelsPublishing(state),
-    isFetchingMyCollections: selectIsFetchingMyCollections(state),
+    isFetchingMyCollectionClaims: selectIsFetchingMyCollectionClaims(state),
     hasMembership: selectHasOdyseeMembership(state),
```

Comment:
Entering http://localhost:9090/@dev-test:8?view=channels results in `resolve` infinite loop.